docs/integrations/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I think you're right to add this, but do you mind opening a separate PR for this change as I might need to involve other reviewers in the PR? (this could delay the merge)

Reply:
Yep can do